2025REDOBuildSeason/src/main/java/frc/robot/Subsystem/FrontLimelight.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.Subsystem;
+
+import frc.robot.Devices.Limelight;
+
+public class FrontLimelight extends Limelight implements Subsystem{
+    
+    private static FrontLimelight instance = null;
+
+    public static FrontLimelight getInstance()
+    {
+        if (instance == null)
+        {
+            instance = new FrontLimelight();
+        }
+        return instance;
+    }
+
+    private FrontLimelight()
+    {   
+        super("limelight-front", 0, "limelight-front", 1); //figure out actual port
+        this.limelightMountAngleDegrees = 0;
+        this.limelightLensHeightMeters = 0;
+        SubsystemManager.registerSubsystem(this);
+    }
+
+    public String getFrontTargetName(){
+        return findTagName();
+    }
+
+    public void update(){
+        limelightUpdate();
+    };
+
+    public void initialize(){
+
+    };
+
+    public void log(){
+        logtoSmartDashboard();
```

Comment:
Keep the consistent casing?
```suggestion
        logToSmartDashboard();
```